UnitTests/CommonTestUtils/SubmoduleTestHelpers.cs

```diff
@@ -13,10 +13,10 @@ public static async Task<SubmoduleInfoResult> UpdateSubmoduleStructureAndWaitFor
             provider.StatusUpdated += ProviderStatusUpdated;
             try
             {
-                provider.UpdateSubmodulesStructure(
-                    workingDirectory: module.WorkingDir,
-                    noBranchText: string.Empty,
-                    updateStatus: updateStatus);
+                await provider.UpdateSubmodulesStructureAsync(
+                        workingDirectory: module.WorkingDir,
+                        noBranchText: string.Empty,
+                        updateStatus: updateStatus).ConfigureAwait(false);
```

Comment:
`ConfigureAwait(false)` doesn't capture the context, and the continuation is performed on the same worker thread thread.
Since we aren't testing the UI here, thread affinity isn't important.

Reply:
I removed all `ConfigureAwait(false)`.